examples/cp4s/README.md

```diff
@@ -0,0 +1,63 @@
+# Test CP4S Module
+
+## 1. Set up access to IBM Cloud
+
+If running this module from your local terminal, you need to set the credentials to access IBM Cloud. You can will also need to set the IC_API_KEY the steps are
+
+1. Make sure you’re logged into cloud.ibm.com and within the correct account.
+2. Go to [https://cloud.ibm.com/iam/apikeys](https://cloud.ibm.com/iam/apikeys) and click on Create an IBM API key
+3. Provide a name and description.
+4. export the key as IC_API_KEY
+
+``` bash
+export IC_API_KEY=<given_secret> # pragma: allowlist secret
+```
+
+You can define the IBM Cloud credentials in the IBM provider block but it is recommended to pass them in as environment variables.
+
+## 2. Test
+
+### Using Terraform client
+
+Follow these instructions to test the Terraform Module manually
+
+Create the file `test.auto.tfvars` with the following input variables, these values are fake examples:
+
+```hcl
+source          = "./.."
+
+// ROKS cluster parameters:
+cluster_config_path = data.ibm_container_cluster_config.cluster_config.config_file_path
+region = var.region
+resource_group_name = var.resource_group_name
+cluster_id = var.cluster_id
+
+// Entitled Registry parameters:
+entitled_registry_key        = var.entitled_registry_key
+entitled_registry_user_email = var.entitled_registry_user_email
+
+admin_user = var.admin_user
+```
+
+These parameters are:
```

Comment:
Add the parameters info in tabular format

reference https://github.com/terraform-ibm-modules/terraform-ibm-satellite/blob/main/.github/workflows/test.yml

Reply:
^ can you check if this is the correct link you wanted me to copy from

Reply:
sorry, this is the link 

https://github.com/terraform-ibm-modules/terraform-ibm-satellite#inputs

Reply:
Fixed